docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -4363,6 +4364,47 @@ image::docs/images/meraki-splash-config.png[scaledwidth="100%",alt="Splash page
 
 NOTE: Even if you enabled the bridge mode on the access point, it will still send a NATed IP address during the initial HTTP request. To workaround this you will need to forward the DHCP traffic of the network your nodes are connected to so PacketFence has the proper IP information of the devices.
 
+CoA Compliance
```

Comment:
What is CoA compliance ?

Not sure this is the right word.

Unless @louismunro can find an RFC for that